ts/src/helpers.cpp

```diff
@@ -279,12 +279,17 @@ std::map<std::string, FREQ_SETTINGS> helpers::parseFrequencies(const std::string
     std::vector<std::string> v = split(sub, ',');
     for (const std::string& xs : v) {
         std::vector<std::string> parts = split(xs.substr(1, xs.length() - 2), '|');
-        if (parts.size() == 3 || parts.size() == 4) {
+        if (parts.size() == 3 || parts.size() == 4 || parts.size() == 5) {
```

Comment:
I mainly kept it for backwards compatibility, so a newer TS plugin version would still be able to parse an older ArmA addon command. Removing the check for `parts.size() == 4` would break the frequency modification for TS if the server is running an outdated version.

I'm happy to remove the additional check if you don't think this backward compatibility is important enough to maintain it.